MSYS2 Access Violation

The 'MSYS2 MinGW 64-bit' GitHub Actions job has started crashing with 'Windows fatal exception: access violation' - https://github.com/python-pillow/Pillow/runs/1160314080?check_suite_focus=true

```
Tests/test_imagetk.py::test_kw PASSED                                    [ 92%]
Windows fatal exception: access violation

Current thread 0x0000069c (most recent call first):
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pillow-8.0.0.dev0-py3.8-mingw.egg/PIL/ImageTk.py", line 197 in paste
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pillow-8.0.0.dev0-py3.8-mingw.egg/PIL/ImageTk.py", line 115 in __init__
  File "D:/a/Pillow/Pillow/Tests/test_imagetk.py", line 61 in test_photoimage
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/python.py", line 184 in pytest_pyfunc_call
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/callers.py", line 187 in _multicall
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/hooks.py", line 286 in __call__
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/python.py", line 1479 in runtest
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/runner.py", line 135 in pytest_runtest_call
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/callers.py", line 187 in _multicall
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/hooks.py", line 286 in __call__
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/runner.py", line 217 in <lambda>
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/runner.py", line 244 in from_call
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/runner.py", line 216 in call_runtest_hook
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/runner.py", line 186 in call_and_report
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/runner.py", line 100 in runtestprotocol
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/runner.py", line 85 in pytest_runtest_protocol
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/callers.py", line 187 in _multicall
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/hooks.py", line 286 in __call__
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/main.py", line 272 in pytest_runtestloop
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/callers.py", line 187 in _multicall
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/hooks.py", line 286 in __call__
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/main.py", line 247 in _main
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/main.py", line 191 in wrap_session
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/main.py", line 240 in pytest_cmdline_main
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/callers.py", line 187 in _multicall
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/manager.py", line 84 in <lambda>
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/manager.py", line 93 in _hookexec
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pluggy/hooks.py", line 286 in __call__
  File "C:/msys64/mingw64/lib/python3.8/site-packages/_pytest/config/__init__.py", line 124 in main
  File "C:/msys64/mingw64/lib/python3.8/site-packages/pytest/__main__.py", line 7 in <module>
  File "C:/msys64/mingw64/lib/python3.8/runpy.py", line 87 in _run_code
  File "C:/msys64/mingw64/lib/python3.8/runpy.py", line 194 in _run_module_as_main
D:\a\_temp\6b7d7523-abb8-41fe-b301-0f3638ee1c61: line 2:   789 Segmentation fault      python3 -m pytest -vx --cov PIL --cov Tests --cov-report term --cov-report xml Tests
Tests/test_imagetk.py::test_photoimage 
Error: Process completed with exit code 139.
```

The last time that the job passed, the 'Set up job' step reported 'Environment: windows-2019 Version: 20200827.1'.
The first time it failed, it reported 'Environment: windows-2019 Version: 20200920.1'.

So I'm guessing that one of these changes is the trigger - https://github.com/actions/virtual-environments/commit/0dab0c0159f67eea5653d0d4a1fa123f3cd4d7ca

--- a/Tests/test_imagetk.py
+++ b/Tests/test_imagetk.py
@@ -2,7 +2,7 @@ import pytest
 
 from PIL import Image
 
-from .helper import assert_image_equal, hopper, is_mingw
+from .helper import assert_image_equal, hopper
 
 try:
     import tkinter as tk
@@ -52,7 +52,6 @@ def test_kw():
     assert im is None
 
 
-@pytest.mark.skipif(is_mingw(), reason="Segmentation fault")
 def test_photoimage():
     for mode in TK_MODES:
         # test as image:

--- a/src/Tk/tkImaging.c
+++ b/src/Tk/tkImaging.c
@@ -63,7 +63,7 @@ ImagingFind(const char* name)
     Py_ssize_t id;
 
     /* FIXME: use CObject instead? */
-#if defined(_MSC_VER) && defined(_WIN64)
+#if defined(_WIN64)
     id = _atoi64(name);
 #else
     id = atol(name);

